File: MyJavaApplication/app/src/main/java/com/example/prana/myjavaapplication/MainActivity.java
package com.example.prana.myjavaapplication;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.util.Log;
import android.view.View;
import android.widget.EditText;
import android.widget.Toast;

public class MainActivity extends AppCompatActivity {

    public void ClickFunction(View view)
    {
        EditText myUsername=(EditText) findViewById(R.id.Username);
        Toast.makeText(this, myUsername.getText().toString(), Toast.LENGTH_SHORT).show();
        EditText myPassword=(EditText) findViewById(R.id.Password);
        Toast.makeText(this, myPassword.getText().toString(), Toast.LENGTH_SHORT).show();
    }
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }
}

File: MyFirstApp/app/src/main/java/com/example/prana/myfirstapp/MainActivity.kt
package com.example.prana.myfirstapp;

import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;
import android.util.Log;
import android.view.View


class MainActivity : AppCompatActivity() {

    public void clickFunction(View view){
    Log.i("Info:","Pressed");
    }


    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
    }
}
